README.md

```diff
@@ -12,10 +21,15 @@ $ makepkg -c
 
 - Add following lines to `/etc/pam.d/system-auth`:
 ```
-auth    required    pam_eye.so    DOMAIN
+auth    optional    pam_eye.so    URL [TIMEOUT_MS [nodebug]]
 ```
 
-Replace placeholder `DOMAIN` with your pam_eye gateway host.
+Replace placeholder `URL` with your **pam_eye** gateway url (without http://).
+
+`TIMEOUT_MS` is the maximum amount of milliseconds which **pam_eye** could take.
+By default it's 200 milliseconds.
+
+`nodebug` option can be used to prevent logging to syslog about faulty servers.
```

Comment:
I agree with you, but there is no debug messages, there is only error messages about serious problems.